src/components/registration/school.svg

```diff
@@ -0,0 +1,84 @@
+		c0,8.882,6.792,15.673,15.673,15.673h469.159c8.882,0,15.673-6.792,15.673-15.673V250.384
+		C499.984,241.502,493.192,234.71,484.31,234.71z M248.686,116.637l81.502,69.486v260.702H166.661v-260.18L248.686,116.637z
+		 M125.388,163.135L248.686,58.122l123.298,105.012l-8.882,9.927l-4.702-4.18c-1.567-1.567-3.135-3.135-5.224-4.18l-94.563-80.457
+		c-3.135-2.612-6.792-3.657-9.927-3.657c-3.657,0-7.314,1.045-9.927,3.657l-95.086,80.98c-2.09,1.045-3.657,2.612-4.702,4.18
+		l-4.702,4.18L125.388,163.135z M30.824,266.057h104.49v181.29H30.824C30.824,447.347,30.824,266.057,30.824,266.057z
+		 M468.637,447.347H361.535v-181.29h107.102V447.347z"/>
+	<path style="fill:#3A2C51;" d="M288.392,350.171h-79.935c-8.882,0-15.673-6.792-15.673-15.673s6.792-15.673,15.673-15.673h79.935
+		c8.882,0,15.673,6.792,15.673,15.673S297.273,350.171,288.392,350.171z"/>
+	<path style="fill:#3A2C51;" d="M59.037,341.812c-8.882,0-15.673-6.792-15.673-15.673v-19.853c0-8.882,6.792-15.673,15.673-15.673
+		s15.673,6.792,15.673,15.673v19.853C74.71,335.02,67.918,341.812,59.037,341.812z"/>
+	<path style="fill:#3A2C51;" d="M106.057,341.812c-8.882,0-15.673-6.792-15.673-15.673v-19.853c0-8.882,6.792-15.673,15.673-15.673
+		s15.673,6.792,15.673,15.673v19.853C121.731,335.02,114.939,341.812,106.057,341.812z"/>
+	<path style="fill:#3A2C51;" d="M59.037,418.612c-8.882,0-15.673-6.792-15.673-15.673v-19.853c0-8.882,6.792-15.673,15.673-15.673
+		s15.673,6.792,15.673,15.673v19.853C74.71,411.82,67.918,418.612,59.037,418.612z"/>
+	<path style="fill:#3A2C51;" d="M106.057,418.612c-8.882,0-15.673-6.792-15.673-15.673v-19.853c0-8.882,6.792-15.673,15.673-15.673
+		s15.673,6.792,15.673,15.673v19.853C121.731,411.82,114.939,418.612,106.057,418.612z"/>
+	<path style="fill:#3A2C51;" d="M392.359,341.812c-8.882,0-15.673-6.792-15.673-15.673v-19.853c0-8.882,6.792-15.673,15.673-15.673
+		s15.673,6.792,15.673,15.673v19.853C408.033,335.02,401.241,341.812,392.359,341.812z"/>
+	<path style="fill:#3A2C51;" d="M439.38,341.812c-8.882,0-15.673-6.792-15.673-15.673v-19.853c0-8.882,6.792-15.673,15.673-15.673
+		c8.882,0,15.673,6.792,15.673,15.673v19.853C455.053,335.02,448.261,341.812,439.38,341.812z"/>
+	<path style="fill:#3A2C51;" d="M392.359,418.612c-8.882,0-15.673-6.792-15.673-15.673v-19.853c0-8.882,6.792-15.673,15.673-15.673
+		s15.673,6.792,15.673,15.673v19.853C408.033,411.82,401.241,418.612,392.359,418.612z"/>
+	<path style="fill:#3A2C51;" d="M439.38,418.612c-8.882,0-15.673-6.792-15.673-15.673v-19.853c0-8.882,6.792-15.673,15.673-15.673
+		c8.882,0,15.673,6.792,15.673,15.673v19.853C455.053,411.82,448.261,418.612,439.38,418.612z"/>
+	<path style="fill:#3A2C51;" d="M249.731,291.657c-22.988,0-41.273-18.286-41.273-41.273s18.286-41.273,41.273-41.273
+		s41.273,18.286,41.273,41.273S272.718,291.657,249.731,291.657z M249.731,240.457c-5.224,0-9.927,4.702-9.927,9.927
+		c0,5.225,4.702,9.927,9.927,9.927s9.927-4.702,9.927-9.927C259.657,245.159,254.955,240.457,249.731,240.457z"/>
+</g>
```

Comment:
so many <g></g> . why is it necessary?

Reply:
I think that it's svg format and it's ok for it